step2-04/demo/README.md

```diff
@@ -1,6 +1,6 @@
 # Step 2.4 - React Context (Demo)
 
-[Lessons](../) | [Exercise](./exercise/) | [Demo](./demo/)
+[Lessons](../../) | [Exercise](../exercise/)
```

Comment:
this is intentional - these links only work in live site / npm if you keep at one level

Reply:
:(